osmaxx/conversion/constants/output_format.py

```diff
@@ -1,7 +1,7 @@
 import uuid
 from collections import OrderedDict, namedtuple
 
-from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
Why do we need gettext_lazy here? Because lazy sounds better? :wink: 
http://stackoverflow.com/questions/4160770/when-should-i-use-ugettext-lazy